src/internal/connector/onedrive/collection.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/microsoftgraph/msgraph-sdk-go/models"
+	"github.com/spatialcurrent/go-lazy/pkg/lazy"
```

Comment:
just checking if this is something we're alright taking a dependency on. Repo on github seems to only have 2 stars and was last updated close to two years ago